dockers/docker-orchagent/start.sh

```diff
@@ -35,14 +35,14 @@ SWSSCONFIG_ARGS="00-copp.config.json "
 
 if [ "$HWSKU" == "Force10-S6000" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
-    SWSSCONFIG_ARGS+="td2.32ports.qos.json td2.32ports.buffers.json "
+    SWSSCONFIG_ARGS+="td2.32ports.buffers.json td2.32ports.qos.json "
```

Comment:
I think by applying buffer configuration after qos configuration we rewrite "PORT_QOS_MAP_TABLE:*: pfc_enable" settings. At least I see there're issues with pfc frame generation.

Sorry. I don't have more information for you.

Do you remember last time I split the buffer/qos configuration files into chunks? https://github.com/Azure/sonic-swss/pull/113/commits/8056a15dac69a1b0b71c62d40df47084c302835f

I did this because it was just one possibility to make it working. That time you told me we're having bugs in saibcm.

So this time I believe we have some bugs with saibcm too.